### ISSUE TITLE
a0 vs a7 for system calls

### ISSUE THREAD

**Issue by TheThirdOne** [andrescv/jupiter#37]
Jupiter uses `a0` for the system call number. Most other simulators/kernels use `a7` for the system call number. Notably, spike with pk, rv8, qemu for userspace, RARS, and RIPES all use `a7`. Venus would be the notable simulator that uses `a0`.

Would you be interested in changing to use a7 instead of a0? I would be willing to make a PR, but wanted to get a "go ahead" 
before I start.

**Comment by andrescv** [comment]
Yes absolutely, in the beginning of the project I thought that register `a0` would be a good choice for system calls because we used to use Venus in classroom. If you could make a PR with that change, go ahead totally agree with the change.
